src/preferences.md

```diff
@@ -50,18 +50,19 @@ deck.
 
 ## Scheduling
 
-**Show learning cards with larger steps before**  
-Only shown when the 2.1 scheduler is enabled. Normally learning cards with a 1+
-day delay are shown after normal reviews. If checked, Anki will show them before
-normal reviews instead.
-
-**Anki 2.1 scheduler**  
-The Anki 2.1 scheduler is documented here:
-<https://faqs.ankiweb.net/the-anki-2.1-scheduler.html>
-
-**Mix new cards and reviews**:
-This drop-down box controls when new cards are shown: either mixed
-with, before, or after all reviews.
+**Show next review time above answer buttons**  
+Useful to know how far in the future your cards are being pushed.
+
+**Show remaining card count review**  
+Disable this option to hide the card count at the bottom of the screen.
+
+**Legacy timezone handling**  
+Please, see this:  
```

Comment:
```suggestion
Please see here:
```

"Please, " places a little too much emphasis on the word please, and sounds like you're begging. :-)